src/xcode/ENA/ENA/Source/Scenes/QRScanner/QRScannerViewModel.swift

```diff
@@ -47,7 +47,8 @@ class QRScannerViewModel: NSObject, AVCaptureMetadataOutputObjectsDelegate {
 	
 	func didScan(metadataObjects: [MetadataObject]) {
 		guard isScanningActivated else {
-			Log.info("Scanning not stopped from previous run")
+			completion(.failure(.scanningError(.scanningDeactivated)))
```

Comment:
Is there anything we can tell the user if this happens? Wouldn't it be a coding error if this happens? Not sure if it makes sense to return an error here 🤔

Reply:
Okay, seems fine for me to show the same error message